src/plugins/data/public/ui/filter_bar/filter_bar.tsx

```diff
@@ -171,12 +173,14 @@ function FilterBarUI(props: Props) {
 
   const classes = classNames('globalFilterBar', props.className);
   const filterBarPrefixText = i18n.translate('data.search.filterBar.filterBarPrefixText', {
-    defaultMessage: 'Filters: ',
+    defaultMessage: 'Filters',
   });
+  const filterGroupClassName =
+    useNewHeader && !props.fixFilterBar ? 'globalFilterGroup--compressed' : 'globalFilterGroup';
```

Comment:
nit: should we just add the compressed class in the compress case vs switching between the two?

Reply:
will make the change